src/lib/fetch.ts

```diff
@@ -91,11 +102,42 @@ export async function azleFetch(input: any, init?: any): Promise<any> {
             text: async () => {
                 return contents.toString();
             }
-        };
+        } as any;
     }
 
-    if (url.protocol === 'http:') {
-        // TODO http call to the management canister
+    if (url.protocol === 'https:' || url.protocol === 'http:') {
+        const body = await prepareRequestBody(init);
+        // TODO also do headers and method and everything like on the client?
+
+        const response = await azleFetch(`icp://aaaaa-aa/http_request`, {
+            body: serialize({
+                args: [
+                    {
```

Comment:
Should we be supported more of the options then this? Are these the same options that we talked about on the front end fetch and decided not to support?

Reply:
If we aren't going to support them should be output warnings if people try to use them?

Reply:
I went through and added the same warning on the backend, there are a couple properties that might exist that we don't have warnings for, but the types are off...I'm willing to let that slide for now. I also added a warning for the window property to the frontend and backend fetch